### ISSUE TITLE
rename repo

### ISSUE THREAD

**Issue by jonschlinkert** [assemble/assemble-example#1]
This repo needs to be called something else. It's a great example of how to get started with assemble, but I think it's difficult to notice with all of the "assemble-*" repos we have.

@Melindrea, any ideas for what to name this? you're good at this! :-)

**Comment by Melindrea** [comment]
assemble-grunt-startingkit? And later we'll get assemble-gulp-startingkit?

**Comment by jonschlinkert** [comment]
good point! I forgot sometimes that most of the examples are specific to grunt. good ideas, thanks!
